From: Ian Jackson <ian.jackson@citrix.com>
Date: Fri, 12 Oct 2018 17:56:56 +0100
X-Dgit-Generated: 4.17.2+76-ge1f9cb16e2-1 eb1d2e52cd28ca77bb08b59998c0eb152e3983f4
Subject: docs/man/xen-vbd-interface.7: Provide properly-formatted NAME section

This manpage was omitted from
   docs/man: Provide properly-formatted NAME sections
because I was previously building with markdown not installed.

Signed-off-by: Ian Jackson <ian.jackson@citrix.com>

---

diff --git a/docs/man/xen-vbd-interface.7.pandoc b/docs/man/xen-vbd-interface.7.pandoc
index ba0d159dfa7e..2f18d5b72e6a 100644
--- a/docs/man/xen-vbd-interface.7.pandoc
+++ b/docs/man/xen-vbd-interface.7.pandoc
@@ -1,3 +1,8 @@
+Name
+----
+
+xen-vbd-interface - Xen paravirtualised block device protocol
+
 Xen guest interface
 -------------------
 
